warehouse/static/sass/blocks/_site-header.scss

```diff
@@ -33,4 +33,9 @@
       max-width: 50px;
     }
   }
+
+  // Test PyPI style
+  &--testing {
+    background: $header-background-color url("../images/testing.png") 0 50%;
```

Comment:
I think we want to be able to cache-bust this image instead. We'll have to specify this in-line in the template so we can use `request.static_path('warehouse:static/dist/images/testing.png')`

Reply:
ok, no problem :)